src/components/pages/profile/profile-form.tsx

```diff
@@ -61,8 +64,8 @@ const ProfileForm: FC<FormProfileProps> = ({ className }) => {
             </FormItem>
           )}
         />
-        <Button className="!mt-8 w-full" type="submit">
-          Submit
+        <Button className="!mt-[5.75rem] w-full md:!mt-8" type="submit">
```

Comment:
If we won't use `space-y-8` on the `form`, `!important` signs can be removed

```suggestion
        <Button className="mt-[5.75rem] w-full md:mt-8" type="submit">
```